tests/integration/census/test_census_household_submission_data.py

```diff
@@ -0,0 +1,1147 @@
+            {
+                "group_instance": 1,
+                "answer_id": "visitor-sex-answer",
+                "block_id": "visitor-sex",
+                "group_id": "visitors",
+                "answer_instance": 0,
+                "value": "Male"
+            },
+            {
+                "group_instance": 1,
+                "answer_id": "visitor-date-of-birth-answer",
+                "block_id": "visitor-date-of-birth",
+                "group_id": "visitors",
+                "answer_instance": 0,
+                "value": "17/10/1985"
+            },
+            {
+                "group_instance": 1,
+                "answer_id": "visitor-uk-resident-answer-other",
+                "block_id": "visitor-uk-resident",
+                "group_id": "visitors",
+                "answer_instance": 0,
+                "value": ""
+            },
+            {
+                "group_instance": 1,
+                "answer_id": "visitor-uk-resident-answer",
+                "block_id": "visitor-uk-resident",
+                "group_id": "visitors",
+                "answer_instance": 0,
+                "value": "Yes, usually lives in the United Kingdom"
+            },
+            {
+                "group_instance": 1,
+                "answer_id": "visitor-address-answer-postcode",
+                "block_id": "visitor-address",
+                "group_id": "visitors",
+                "answer_instance": 0,
+                "value": "12345"
+            },
+            {
+                "group_instance": 1,
+                "answer_id": "visitor-address-answer-street",
+                "block_id": "visitor-address",
+                "group_id": "visitors",
+                "answer_instance": 0,
+                "value": ""
+            },
+            {
+                "group_instance": 1,
+                "answer_id": "visitor-address-answer-building",
+                "block_id": "visitor-address",
+                "group_id": "visitors",
+                "answer_instance": 0,
+                "value": "1009"
+            },
+            {
+                "group_instance": 1,
+                "answer_id": "visitor-address-answer-city",
+                "block_id": "visitor-address",
+                "group_id": "visitors",
+                "answer_instance": 0,
+                "value": "Detroit"
+            },
+            {
+                "group_instance": 1,
+                "answer_id": "visitor-address-answer-county",
+                "block_id": "visitor-address",
+                "group_id": "visitors",
+                "answer_instance": 0,
+                "value": ""
+            }
+        ]
+
+        return expected_downstream_data
+
+    def complete_survey(self, eq_id, form_type_id):
+        variant_flags = {'sexual_identity': 'false'}
+        self.launchSurvey(eq_id, form_type_id, region_code='GB-ENG', variant_flags=variant_flags, roles=['dumper'])
+
+        self.who_live_here_section()
```

Comment:
I think these method names should be prefixed with `complete_`, so this would be `complete_who_lives_here_section` to make it clearer what the method is doing.
Also, it should be lives rather than live.